consensus/ibft/timeout.go

```diff
@@ -6,15 +6,16 @@ import (
 )
 
 const (
-	baseTimeout = 10 * time.Second
-	maxTimeout  = 300 * time.Second
+	maxTimeoutMultiplier = 30
 )
 
 // exponentialTimeout calculates the timeout duration in seconds as exponential function
-// where maximum value returned can't exceed 300 seconds
-// t = 10 + 2^exponent	where exponent > 0
-// t = 10				where exponent = 0
-func exponentialTimeout(exponent uint64) time.Duration {
+// where maximum value returned can't exceed 30 * baseTimeout
+// t = baseTimeout * maxTimeoutMultiplier where exponent > 8
+// t = baseTimeout + 2^exponent	          where exponent > 0
+// t = baseTimeout				                where exponent = 0
```

Comment:
Major nitpick: please use tabs to separate out `where`, it's uneven now 🙈

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/582/commits/18015d970fc2e41e28938418c4ccea5f85b0f961